services/generic/scene_control.md

```diff
@@ -1,21 +1,46 @@
-### Scene controller service
+# Scene Control Service
 
-The service represents a device which can be used to control scenes. Normally it's remote controller.
+The service represents a device which can be used to control scenes, such as a remote controller.
 
-#### Service names
+## Service name
 
 `scene_ctrl`
 
-#### Interfaces
+## Interfaces
 
-Type | Interface            | Value type | Description
------|----------------------|------------|-------------------
-in   | cmd.scene.get_report | null       | The command is a request for current scene.
-in   | cmd.scene.set        | string     | Set scene
-out  | evt.scene.report     | string     | Event is generated whenever scene button is pressed on controller.
+| Type | Interface            | Value type | Description                                                            |
+|------|----------------------|------------|------------------------------------------------------------------------|
+| in   | cmd.scene.get_report | null       | Requests the currently active scene.                                   |
+| in   | cmd.scene.set        | string     | Sets the scene to the one provided in the value.                       |
+| out  | evt.scene.report     | string     | Event is generated whenever scene button is pressed on the controller. |
 
-#### Service props
+> Devices designed only as scene controllers may support only `evt.scene.report` interface and send report whenever a button is pressed.
```

Comment:
"...whenever a physical button is pressed."(?)

Reply:
Ok